### PR TITLE
Bring Back Rails 5.0 Support

### PR DESCRIPTION
**Pull request by amatsuda**
This patch brings Rails 5.0 support back, with a little bit of code complexity in the testing script.

In my opinion the complexity is an acceptable trade-off for the wider versions support, but I'd like to hear @kou's opinion if I'm in the right direction.

### CODE REVIEW

File: test/test_app.sh [hunk 1]
```diff
@@ -33,18 +33,32 @@ end
...
   rails webpacker:install
 fi
 rails generate model item name:string
 sed -i'' -e 's/# //g' test/models/item_test.rb
 rails db:migrate
-rails test -v | tee test.log
-grep models: test.log
-grep assertions test.log
+
+if [ `echo "${RAILS_VERSION} >= 5.1" | bc` = 1 ]; then
```
Comment (kou) [inline test/test_app.sh:53]:
```suggestion
if [ $(echo "${RAILS_VERSION} >= 5.1" | bc) -eq 1 ]; then
```

**Review by kou — APPROVED** [review]
+1

File: test/test_app.sh (hunk 1 above)
Comment (kou) [reply test/test_app.sh:53]: * `$(...)` is better than `` `...` `` in shell
* `=` is a string operator and `-eq` is a number operator in `[`

Comment (amatsuda) [reply test/test_app.sh]: Thank you for the review. Applied your suggestions and force-pushed.

**Comment by kou** [comment]
Can we use squash merge for this?
If you prefer rebase merge, we can enable it and use it.

**Comment by amatsuda** [comment]
I strongly prefer rebase merge over squash merge, since squashing the original commits feels like doing something disrespectful to the commit authors (and to the commit objects), I mean, not only about my PRs but in general.
I appreciate if you could enable squash merging (and encourage the use of it).

**Comment by kou** [comment]
OK.
I enabled rebase merge and used.

**Comment by amatsuda** [comment]
🙏

**Pull request merged**